app/_data/konnect_oas_data.json

```diff
@@ -170,6 +170,27 @@
       }
     ]
   },
+  {
+    "id": "f3675942-dbc0-4298-bec2-cbcfdcc10fab",
+    "title": "Konnect Mesh Manager",
+    "latestVersion": {
+      "name": "v1",
+      "id": "0426ee88-bf76-45e3-9f89-fc03bea0c3a5"
```

Comment:
This wasn't in the UI (that I could see), so I used the API product spec to find it. Could someone double check that this is correct?

Reply:
It's right. You can grab it from the URL:

```
https://cloud.konghq.com/us/api-products/f3675942-dbc0-4298-bec2-cbcfdcc10fab/versions/0426ee88-bf76-45e3-9f89-fc03bea0c3a5
```
Where the last ID after `versions/` is the one you want.

Reply:
URL is the only place I've seen version specified, you got it right